packages/cdk-graph-plugin-threat-composer/src/model-generator/base-model/README.md

```diff
@@ -0,0 +1,9 @@
+## Modifying the Base Threat Model
+
+This folder contains a base threat model (`.tc.json` file) with example threats mapped to CDK Nag rules, and the plugin filters to only the applicable threats to the CDK project.
+
+The easiest way to modify the threat model is to import the json file in the [Threat Composer Website](https://awslabs.github.io/threat-composer/), modify it using the UI, and then export it as json and replace the contents in this repository.
```

Comment:
"is to import the json file into the [Threat Composer](https://awslabs.github.io/threat-composer/) tool"